packages/react-core/src/components/Text/examples/TextPlainList.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Text, TextVariants, TextContent, TextList, TextListVariants, TextListItem } from '@patternfly/react-core';
+
+export const TextOrderedList: React.FunctionComponent = () => (
+  <TextContent>
+    <Text component={TextVariants.h3}>Plain unordered list</Text>
+    <TextList isPlain>
+      <TextListItem>In fermentum leo eu lectus mollis, quis dictum mi aliquet.</TextListItem>
+      <TextListItem>Morbi eu nulla lobortis, lobortis est in, fringilla felis.</TextListItem>
+      <TextListItem>
+        Aliquam nec felis in sapien venenatis viverra fermentum nec lectus.
+        <TextList>
+          <TextListItem>In fermentum leo eu lectus mollis, quis dictum mi aliquet.</TextListItem>
+          <TextListItem>Morbi eu nulla lobortis, lobortis est in, fringilla felis.</TextListItem>
+        </TextList>
```

Comment:
Could maybe omit this nested list, or add some verbiage noting that a non-plain list can eb nested inside a plain list.

Reply:
Good call, removed it - I don't think it's necessary. We have nested examples in the core docs in case we need a reference to that.